cli/src/main/scala/com/gu/pfi/cli/ingestion/CliFileWalker.scala

```diff
@@ -10,27 +9,27 @@ import scala.jdk.StreamConverters._
 import scala.util.Try
 
 class CliFileWalker(enrich: Path => IngestionFile) extends Logging {
-  def walk(path: Path, root: Uri, languages: List[Language]): LazyList[OnDiskFileContext] = {
+  def walk(path: Path, root: Uri, languages: List[Language]): List[OnDiskFileContext] = {
     walk(path, List(root.chain(path.getFileName.toString), root), root, languages)
   }
 
-  private def walk(thisPath: Path, parents: List[Uri], root: Uri, languages: List[Language]): LazyList[OnDiskFileContext] = {
+  private def walk(thisPath: Path, parents: List[Uri], root: Uri, languages: List[Language]): List[OnDiskFileContext] = {
     val enrichedPath = enrich(thisPath)
 
-    OnDiskFileContext(enrichedPath, parents, root.value, languages, thisPath) #:: {
+    OnDiskFileContext(enrichedPath, parents, root.value, languages, thisPath) +: {
       if (Files.isDirectory(thisPath)) {
         Try {
           val fileStream = Files.list(thisPath)
           // eagerly evaluate the file stream; allowing it to be closed to avoid resource exhaustion
-          val evaluated = fileStream.toScala(LazyList)
+          val evaluated = fileStream.toScala(LazyList).toList
```

Comment:
Whoops sorry! FWIW I think you don't need to change the signature entirely, just force the evaluation to a list here

```suggestion
          val evaluated = LazyList.from(fileStream.toScala(List))
```